### PR TITLE
Re-adds airlock charges

### PR DESCRIPTION
**Pull request by Amelia0010**
TG fucking first made them OP, then removed them 🤔

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1081,11 +1081,11 @@
...
 		detonated = 1
 		charge = null
-		for(var/mob/living/carbon/human/H in orange(2,src))
+		for(var/mob/living/carbon/human/H in orange(1,src)) //yogs - balance change
 			H.Unconscious(160)
-			H.adjust_fire_stacks(20)
+			H.adjust_fire_stacks(1) //yogs - balance change
 			H.IgniteMob() //Guaranteed knockout and ignition for nearby people
-			H.apply_damage(40, BRUTE, "chest")
+			H.apply_damage(20, BRUTE, "chest") //yogs - balance change
```
Comment (simonset55) [inline code/game/machinery/doors/airlock.dm:1088]: Too op, change it to 80 brute damage to balance it out

File: yogstation.dme [hunk 2]
```diff
@@ -2554,4 +2554,5 @@
 #include "yogstation\code\modules\clothing\clothing.dm"
 #include "yogstation\code\modules\libvg\utf8.dm"
 #include "yogstation\code\modules\spells\spells.dm"
+#include "yogstation\code\modules\uplink\uplink_item.dm"
```
Comment (simonset55) [inline yogstation.dme:2557]: It should be yogstation\code\modules\uplink\uplink_item.dmihåndbold2018

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -1081,11 +1081,11 @@
 		charge.ex_act(EXPLODE_DEVASTATE)
```
Comment (simonset55) [inline code/game/machinery/doors/airlock.dm:1081]: It should be EXPLODE_DEMOLISH, the word devestate could trigger potential vietnam PTSD when we dropped a bunch of napalm on those chings in that village. My god... the children are still screaming in my dreams.

File: yogstation/code/modules/uplink/uplink_item.dm [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/datum/uplink_item/stealthy_weapons/door_charge
+	name = "Explosive Airlock Charge"
+	desc = "A small, easily concealable device. It can be applied to an open airlock panel, booby-trapping it. \
+			The next person to use that airlock will trigger an explosion, knocking them down and destroying \
+			the airlock maintenance panel."
+	item = /obj/item/device/doorCharge
+	cost = 2
```
Comment (simonset55) [inline yogstation/code/modules/uplink/uplink_item.dm:7]: Make the cost 0.1 because it's fucking shit and needs to die lol

**Review by simonset55 — CHANGES_REQUESTED** [review]
Yes

File: yogstation/code/modules/uplink/uplink_item.dm (hunk 4 above)
Comment (Amelia0010) [reply yogstation/code/modules/uplink/uplink_item.dm:7]: Salt much?

**Review by ThatLing — APPROVED** [review]

**Pull request merged**